server/models/services/ModelService.js

```diff
@@ -0,0 +1,216 @@
+const { sequelize } = require('../');
+
+// todo: convert to class structure
+
+/**
+ * Created Query Sequelize Example:
+ * UserModel.findOne({
+ *   where: { id },
+ *   attributes: [ 'username', email ],
+ *   include: [
+ *     {
+ *       model: RoleModel,
+ *       as: 'roles'
+ *       attributes: [ 'name', 'permissions' ]
+ *     }
+ *   ]
+ * })
+ *
+ * @example
+ * // returns { attribute: 'value', associationData: [ { associationAttribute: 'associationAttributeValue' } ] }
+ * return ModelService.getById(Model, id, [ 'attribute' ], [
+ *   {
+ *     model: AssociationModel,
+ *     as: 'associationData',
+ *     attributes: [ 'associationAttribute' ]
+ *   }
+ * ]
+ *
+ * @param {Model} model - Sequelize Model instance to query for
+ * @param {number | string} id - ID of the Sequelize Model to query for
+ * @param {any[]} attributes - attributes of the Sequelize Model to be returned
+ * @param {any[]} include - information on Sequelize Model relationships
+ * @returns {Promise<Model>} - Sequelize Model
+ */
+const getById = async (model, id, attributes = [], include = []) => {
+    if (!model) throw new Error('Model not defined'); // TODO: pass through custom APIError
+
+    // findByPk
+    return await model.findOne({
+        where: { id },
+        attributes,
```

Comment:
I actually don't think we need the ability to request specific attributes! GraphQL handles removing unused fields internally so it wouldn't be convenient (it would be difficult) to pull the request attributes out a GraphQL request.

When I say this, I'm not talking about associations, I'm specifically referring to all the attributes which are part of the model's table. I think this service should load all of them, every time, no need to provide a specific list.